plugin.go

```diff
@@ -154,10 +159,20 @@ func (p Plugin) CheckAllowed(ip string) (bool, string, error) {
 	var allowed bool
 	for _, allowedCountry := range p.allowedCountries {
 		if allowedCountry == country {
-			allowed = true
-			break
+			// allowed = true
+			return true, country, nil
+			//break
```

Comment:
I think the comments can be removed?

```suggestion
			return true, country, nil
```